### PR TITLE
Update Angular2 version to 2.2.3

### PR DESCRIPTION
**Pull request by psantos10**
Update ng to use Angular version 2.2.3 instead of 2.2.1

### CODE REVIEW

**Comment by psantos10** [comment]
I signed it!

**Comment by grizzm0** [comment]
@hansl Has the issue that made us lock the version been fixed? In that case we could change to `^2.2.3`

**Comment by sarunint** [comment]
@grizzm0 I believe so. See angular/angular@64bd672e3af793e210d876244869cb42b007e29e

**Comment by psantos10** [comment]
@sarunint can you check the @googlebot  reply please?

**Comment by sarunint** [comment]
OK

**Comment by sarunint** [comment]
@googlebot I'm OK with the commit

**Comment by sarunint** [comment]
@psantos10 I'm not sure why @googlebot is not responding.

**Comment by minhoryang** [comment]
@sarunint Did you sign the CLA at https://cla.developers.google.com/ ?

**Comment by hansl** [comment]
Googlebot is confused because there are multiple commits by different authors here. Can you squash to just @psantos10 please?

**Comment by hansl** [comment]
Otherwise I'll just make the PR myself.

**Comment by psantos10** [comment]
@hansl squashed

**Comment by hansl** [comment]
@grizzm0 fix the version. We are working on making loose versions work again, but in the meantime we have to carefully update.

File: package.json [hunk 1]
```diff
@@ -41,9 +41,9 @@
   "homepage": "https://github.com/angular/angular-cli",
   "dependencies": {
     "@angular-cli/ast-tools": "^1.0.0",
-    "@angular/compiler": "2.2.1",
-    "@angular/compiler-cli": "2.2.1",
-    "@angular/core": "2.2.1",
+    "@angular/compiler": "^2.2.3",
```
Comment (hansl) [inline package.json]: Lock the version please. Here and everywhere.

**Comment by hansl** [comment]
@psantos10 thanks for the squash. GoogleBot is happier :) Just need to fix the version and I'll get this in. Cheers!

File: package.json (hunk 1 above)
Comment (psantos10) [inline package.json]: Done

**Pull request merged**